src/coordinator/users.js

```diff
@@ -83,6 +93,43 @@ class Users {
     );
   }
 
+  /**
+   * Adds a role to a user
+   *
+   * API Endpoint: '/users/:userId/roles/:roleId'
+   * Method: GET
+   *
+   * @param {string} userId The ID of the user
+   * @param {string} roleId The ID of the role
+   *
+   * @returns {Promise}
+   * @fulfill {ContxtUserRole[]} The newly created user role
```

Comment:
Should this just be a single role or an array of roles?

Reply:
Single, thanks!